rebase-helper.spec

```diff
@@ -98,8 +98,8 @@ PYTHONPATH=$(pwd) py.test-3 -v %{pkgname}
 
 
 %changelog
-* Wed Jul 31 2019 Nikola Forró <nforro@redhat.com> - 0.17.0-1
```

Comment:
Shouldn't this stay in the changelog as well?

Reply:
Why? It was never published.

Reply:
Ah, thanks